scout/adapter/mongo/variant.py

```diff
@@ -292,6 +292,52 @@ def variant(
 
         return variant_obj
 
+    def overlapping_sv_variant(self, case_id, variant_obj):
+        """Returns a SV for a case that is as similar as possible to a SV from another case
+
+        Args:
+            case_id (str): case id for the variant query
+            variant_obj (dict): a variant dictionary from another case
+
+        Returns:
+            hit (Variant): a variant object dictionary
+        """
+        coordinate_query = self.sv_coordinate_query(
+            {
+                "chrom": variant_obj["chromosome"],
+                "start": variant_obj["position"],
+                "end": variant_obj["end"],
+            }
+        )
+        query = {
+            "case_id": case_id,
+            "category": variant_obj["category"],  # sv
+            "variant_type": variant_obj["variant_type"],  # clinical or research
+            "sub_category": variant_obj["sub_category"],  # example -> "del"
+            "$and": coordinate_query,  # query for overlapping SV variants
+        }
+
+        overlapping_svs = list(
+            self.variant_collection.find(
+                query,
+            )
+        )
+        if not overlapping_svs:
+            return None
+        if len(overlapping_svs) == 1:
+            return overlapping_svs[0]
+
+        # If more than one SV is overlapping with this variant
+        # return the one with most similar size
```

Comment:
It ended up very different from the [loqus algorithm](https://github.com/moonso/loqusdb/blob/a6b7ee426357dd493eb31777ded4d6246753c484/loqusdb/plugins/mongo/structural_variant.py#L232). If you don't like it @dnil I can use the code in LoqusDB instead. The thing is, it's not really an exact match so any criterion is good as long as one like it..

Reply:
😄 Indeed, you will find quite a handful of others over time in SVDB, as well as in the loqusdb project. I think the salient point is having some agreement between # found and the number reported.

This works for me. Anything with a lambda in it is already very high up on my liking, and reusing the existing scout overlap definition is very nice. To get the exact same result as loqusdb with the loqusdb code you also need to be using the data from its clustering. It will also create a slightly uncomfortable dependency. 
The results should be very similar when it comes to del / dup recall, but will have some issues with BNDs on different chr, potentially also with inversions.